contrib/action_recognition/README.md

```diff
@@ -1,28 +1,41 @@
 # Action Recognition
 
-This is a place holder. Content will follow soon.
+This folder contains projects relevant to video-based action recognition.
 
-![](./media/action_recognition.gif)
+![](./media/action_recognition.gif "Example of action recognition")
 
 *Example of action recognition*
 
 ## Overview
 
 | Folders |  Description |
 | -------- |  ----------- |
-| [i3d](i3d)  | Scripts for fine-tuning a pre-trained Two-Stream Inflated 3D ConvNet (I3D) model on the HMDB-51 dataset
+| [i3d](i3d)  | Scripts for fine-tuning a pre-trained Two-Stream Inflated 3D ConvNet (I3D) model on the [HMDB-51 dataset](http://serre-lab.clps.brown.edu/resource/hmdb-a-large-human-motion-database/)
```

Comment:
As description I would probably use the exact same sentence for i3d and r2p1d, just with the name i3d/r2p1d changed